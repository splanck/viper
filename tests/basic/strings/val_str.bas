10 PRINT VAL("42")
20 PRINT STR$(42.5)
