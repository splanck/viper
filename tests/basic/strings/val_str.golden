42
42.5
